docs/src/layouts/links.js

```diff
@@ -527,6 +527,11 @@ export default [
       { id: 'PHOTOGRAPHY', route: routes.PHOTOGRAPHY, children: 'Photography' },
       { id: 'SHADOWS', route: routes.SHADOWS_GUIDELINES, children: 'Shadows' },
       { id: 'COPYWRITING', route: routes.COPYWRITING, children: 'Copywriting' },
+      {
```

Comment:
Can this be moved to above copywriting? This is to maintain the new guidelines page order 😄